test/common/testing_idp.go

```diff
@@ -31,7 +31,7 @@ const (
 	defaultNumberOfDedicatedAdmins = 2
 	defaultSecret                  = "rhmiForeva"
 	DefaultTestUserName            = "test-user"
-	defaultPassword                = "Password1"
+	defaultPassword                = "Apollo 440 cant Stop the Rock"
```

Comment:
I am not sure if it is wise to have the password disclosed like this. Instead of relying on default value make sure the proper value is passed as parameter

Reply:
sorry, reverted